expression/integration_test.go

```diff
@@ -2721,12 +2721,6 @@ func (s *testIntegrationSuite) TestCompareBuiltin(c *C) {
 	tk.MustExec("create table t(a bigint unsigned)")
 	tk.MustExec("insert into t value(17666000000000000000)")
 	tk.MustQuery("select * from t where a = 17666000000000000000").Check(testkit.Rows("17666000000000000000"))
-
-	tk.MustExec("drop table if exists t")
```

Comment:
comment it and leave a TODO ?

Reply:
We already have an issue about it. So re-open the issue is enough.